src/ontology/mondo-edit.obo

```diff
@@ -277936,15 +277936,26 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 id: MONDO:0012243
 name: B-cell immunodeficiency, distal limb anomalies, and urogenital malformations
 subset: clingen {source="MONDO:CLINGEN"}
+subset: gard_rare {source="GARD:22284", source="MONDO:GARD"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disorder {source="Orphanet:567502"}
+subset: orphanet_rare {source="Orphanet:567502"}
+subset: rare
 synonym: "B-cell immunodeficiency, distal limb anomalies, and urogenital malformations" EXACT [OMIM:609296]
-synonym: "Bilu syndrome" RELATED [OMIM:609296]
-synonym: "Hoffman syndrome" RELATED [OMIM:609296]
+synonym: "B-cell immunodeficiency-limb anomaly-urogenital malformation syndrome" EXACT [MONDO:0035454]
+synonym: "BILU syndrome" EXACT [OMIM:609296, Orphanet:567502]
```

Comment:
BILU is an acronym. Since the full synonyms is BILU syndrome I did not add the type acronym, but BILU should not be in sentence case. See https://omim.org/entry/609296, "BILU syndrome is an autosomal dominant..." or https://pubmed.ncbi.nlm.nih.gov/15521984/